File: deno/src/nuxt/seo/seo.service.ts
import {
  findOneDomain,
} from "/gen/base/domain/domain.dao.ts";

import {
  findOneSeo,
} from "/gen/nuxt/seo/seo.dao.ts";

/**
 * 查找默认的SEO优化
 */
export async function findDefaultSeo(
  options?: {
    is_debug?: boolean;
  },
): Promise<SeoModel | undefined> {
  
  const seo_model = await findOneSeo(
    undefined,
    undefined,
    options,
  );
  
  return seo_model;
}

File: rust/app/nuxt/seo/seo_service.rs
use color_eyre::eyre::Result;

use generated::common::context::Options;

/// 查找默认的SEO优化
pub async fn find_default_seo(
  options: Option<Options>,
) -> Result<Option<generated::nuxt::seo::seo_model::SeoModel>> {
  
  let seo_model = generated::nuxt::seo::seo_dao::find_one_seo(
    None,
    None,
    options,
  ).await?;
  
  Ok(seo_model)
}
